### ISSUE TITLE
0.9.0 fails to work on cargo crate

### ISSUE THREAD

**Issue by matthiaskrgr** [RustSec/cargo-audit#132]
Running cargo audit 0.8.1 worked just fine on the cargo git repo:
````
    Fetching advisory database from `https://github.com/RustSec/advisory-db.git`
      Loaded 44 security advisories (from /home/matthias/.cargo/advisory-db)
    Scanning Cargo.lock for vulnerabilities (141 crate dependencies)
     Success No vulnerable packages found
````
but 0.9.1 errors:
````
    Fetching advisory database from `https://github.com/RustSec/advisory-db.git`
      Loaded 48 security advisories (from /home/matthias/.cargo/advisory-db)
error: Couldn't load Cargo.lock: parse error: parse error: couldn't find root package
````
It's also strange that the number of advisories differs between cargo-audit versions..?

**Comment by tarcieri** [comment]
Thanks! I managed to reproduce this (ironically I had tested on the `cargo` repo prior to a release, but on a stale `Cargo.lock` which worked at the time)

> It's also strange that the number of advisories differs between cargo-audit versions..?

The new release indexes [advisories filed against core components e.g. `std`](https://github.com/RustSec/cargo-audit/issues/46)

**Comment by tarcieri** [comment]
@matthiaskrgr I've just released v0.9.1 which fixes the problem locally. Can you test it out and confirm it's fixed for you as well?

**Comment by matthiaskrgr** [comment]
Can confirm 0.9.1 works on cargo now, thanks!
